SignalMessaging/environment/OWSSounds.h

```diff
@@ -0,0 +1,97 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, OWSSound) {
```

Comment:
Is there a reason to write all this new code in objc vs swift?

Reply:
I wasn't 100% sure this wouldn't need to move to SSK.  It doesn't look like it needs to.